framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1699,6 +1702,27 @@ public AnnotatedDeclaredType getSelfType(Tree tree) {
     }
 
     /**
+     * Get the AnnotatedDeclaredType for a classTree without kicking off dataflow or calling
```

Comment:
No, postProcessClassTree is not called when this method is called directly. Dataflow is only kicked off the first time addComputedAnnotations is called.  So, dataflow still might be started by this method, so I removed the references to dataflow from these comments.